packages/vite-plugin-cloudflare/playground/node-compat/__tests__/worker-warnings/warnings.spec.ts

```diff
@@ -1,15 +1,18 @@
 import dedent from "ts-dedent";
 import { expect, test, vi } from "vitest";
-import { serverLogs } from "../../../__test-utils__";
+import { isBuild, serverLogs } from "../../../__test-utils__";
 
-test("should display warnings if nodejs_compat is missing", async () => {
-	await vi.waitFor(async () => {
-		expect(serverLogs.warns.join("").replaceAll("\\", "/")).toContain(
-			dedent`
+test.skipIf(isBuild)(
```

Comment:
We I think is that because the node compat worker-warnings playground doesn't have nodejs_compat enabled (in order to trigger the warnings) it does build successfully by externalising the node compat imports. This means that when the build output is previewed, Miniflare fails to start because the node imports can't be resolved.

Something to do with our testing setup means that this doesn't happen consistently so we are going to skip this test for builds for now.